jdaviz/configs/imviz/tests/test_astrowidgets_api.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+import pytest
+from astropy import units as u
+from astropy.io import fits
+from astropy.wcs import WCS
+from numpy.testing import assert_allclose
+
+
+class TestAstrowidgetsAPI:
+    @pytest.fixture(autouse=True)
+    def setup_class(self, imviz_app):
```

Comment:
Note: This is basically the same test setup in #630 . When we get back to the other PR, will have to see how to consolidate this to avoid code duplication.